cfme/tests/integration/test_aws_iam_auth_and_roles.py

```diff
@@ -24,10 +21,7 @@ def pytest_generate_tests(metafunc):
     parameter_list = []
     id_list = []
     # TODO: Include SSUI role_access dict and VIASSUI context
-    role_access_ui = VersionPicker({
-        Version.lowest(): role_access_ui_59z,
-        '5.10': role_access_ui_510z
-    }).pick(appliance.version)
+    role_access_ui = role_access_ui_510z
```

Comment:
This become obsolete with the removal of `VersionPicker`. I think it would be better for readability in the future if we just used `role_access_ui_510z` in our code.

Reply:
Gotcha, I have an assumption here that we may have a different RBAC mapping in 5.11, since there are deprecated features that will result in UI page exclusion. That's why I left the assignment like this, so its easy to re-import and inject the new role_access_ui_511z.

Thoughts? If you'd rather, I can change this to be used directly in the test generation. Wanted to explain my thought process. :)

Reply:
I see your point now. I feel that you stopped kind of half-way by removing `VersionPicker` but leaving the assignment there, if that makes sense. Maybe put a comment in there explaining that it's waiting for `role_access_ui_511z`?

Reply:
That pretty much describes what happened! :) I almost kept tearing code away, but realized it'll likely be version dependent again soon, so I stopped halfway.

Reply:
Note I'm dropping the definitions of `role_access_ui_58z` and `role_access_ui_59z` from cfme/roles.py